openerp/models.py

```diff
@@ -2160,6 +2160,14 @@ def read_group(self, cr, uid, domain, fields, groupby, offset=0, limit=None, con
             for d in fetched_data:
                 d.update(data_dict[d['id']])
 
+        selectionfields = [gb['field'] for gb in annotated_groupbys if gb['type'] == 'selection']
+        if selectionfields:
+            selections_values = {}
+            for selection in selectionfields:
+                selections_values[selection] = dict(self._fields[selection].selection)
```

Comment:
The selection attribute of selection fields can be a callable (function) or the name of a method. And it's not translated when accessing the attribute directly.

I *think* `_description_selection(env)` on the selection field is for reifying the selection items (calling the method/function if any, and translating the label) but check with @rco-odoo to make sure.